src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBagInterface.php

```diff
@@ -18,6 +18,9 @@
  * ParameterBagInterface is the interface implemented by objects that manage service container parameters.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @method array allDeprecated() Returns all deprecated parameters. Not implementing it is deprecated since Symfony 6.2.
+ * @method deprecate(string $name, string $package, string $version, string $message) Deprecates a parameter. Not implementing it is deprecated since Symfony 6.2.
```

Comment:
do we really need to make this part of the interface? I'd prefer if it could stay on ParameterBag

Reply:
`ContainerBuilder` exposes now the only public API.